test-utils/src/build_utils/mod.rs

```diff
@@ -212,6 +223,39 @@ impl Processor {
         }
     }
 
+    fn process_schema_derivation(&mut self) {
+        self.write_mod_entry();
+        let mut write_file = write_test_file(&self.file_name, &self.out_dir);
+        self.write_schema_derivation_header(&write_file);
+        let test_file = crate::parse_schema_derivation_yaml_file(self.entry.path()).unwrap();
+        match test_file {
+            SchemaDerivationYamlTestFile::Single(_) => {
+                write!(
+                    write_file,
+                    include_str!("./templates/schema_derivation_test_body_template"),
+                    name = &self.file_name.replace(".rs", ""),
```

Comment:
The name here is different from the below match for the multiple files.  Seems to me like these should be consistent in using the description for the name.

Reply:
Ah actually, I did this on purpose (but we can possibly rectify it). The reason is that the "single" tests represent the spec query tests, where the file name is something like "correctness_test_1_1". The file itself doesn't contain a description, because the entire file is a single test definition, rather than a list of tests.

So in essence, the filename is the description. I could add them in as placeholders if you think it would make the code more readable, however.